Directory.Build.props

```diff
@@ -143,6 +143,8 @@
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <PackageVersion>$(DotNetVersionBand)-dev</PackageVersion>
     <PackageOutputPath>$(MSBuildThisFileDirectory)artifacts</PackageOutputPath>
+    <LicenseFile>$(MSBuildThisFileDirectory)LICENSE.TXT</LicenseFile>
+    <PackageThirdPartyNoticesFile>$(MSBuildThisFileDirectory)THIRD-PARTY-NOTICES.TXT</PackageThirdPartyNoticesFile>
```

Comment:
Does this include License and TPN too many places? Graphics, Blazor, Blazor (WPF), Blazor (FORMS), etc...

Reply:
I'd rather over include than under include. This is fine as far as I'm concerned. If we make it more complicated and try to slice and dice things we'll probably end up missing too many places either now or eventually.